lib/routes/security_pin/security_and_backup/backup_tiles/backup_provider_tile.dart

```diff
@@ -112,7 +111,7 @@ class _BackupProviderTileState extends State<BackupProviderTile> {
     final backupBloc = AppBlocsProvider.of<BackupBloc>(context);
     return await backupBloc.backupStateStream.first.then(
       (backupState) async {
-        if (backupState != BackupState.start() && previousProvider.isGDrive) {
+        if (backupState != BackupState.start() || previousProvider.isGDrive) {
```

Comment:
@ubbabeck I don't understand this change, what was this addressing? This condition was to prevent showing this dialog on new accounts that are setting Google Drive for the first time. If this is causing issues on iOS or any other condition, we might want to add that condition, 'c' to this clause, instead of changing this:

Let's consider this (a && b), instead of (a || b), we should have ((a && b) || c).

Reply:
The reason for this change was because is so when the user switches from remote storage or iCloud back to GDrive they where prompted to sign out of their gDrive account, which is not necessary the wished behaviour.

do we wish it to be  ?
```suggestion
        if ((backupState != BackupState.start() && previousProvider.isGDrive) ||   previousProvider.isGDrive) {
```

Reply:
Yes, we want to give the users the ability to switch their Google accounts associated with Breez. If they select no, they continue with their previously logged in account.

But, this can be confusing, what we should do imo is sign out at all times if user switches to a non Google Drive provider. What do you think?